capa/features/extractors/ida/insn.py

```diff
@@ -7,9 +7,11 @@
 # See the License for the specific language governing permissions and limitations under the License.
 from typing import Any, Dict, Tuple, Iterator
 
-import idc
+import ida_ua
 import idaapi
 import idautils
+import idc
```

Comment:
I have ordered the imports alphabetically

Reply:
we use isort to ensure imports are ordered by line length

Reply:
I was surprised there was no linter to sort the imports, I didn't realize they were ordered by line length  :sweat_smile: I'll change it, thanks @williballenthin!